src/get-started/flutter-for/android-devs.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Flutter for Android Developers
+description: Learn how to apply Android developer knowledge when learning Flutter.
```

Comment:
In the `description`, "learn" and "learning" seems redundant. How about:

> Learn how to apply Android developer knowledge when building Flutter apps.

If you agree, consider also updating the description of the other "Flutter for ..." pages.